app/src/renderer/vuex/modules/session.js

```diff
@@ -116,8 +142,8 @@ export default () => {
           break
         default:
           // local keyStore
-          state.account = account // TODO: why do we have state.account and state.accounts ??
-          accountAddress = (await state.externals.loadKeys()).find(({ name }) => name === account).address
+          state.localKeyName = localKeyName // TODO: why do we have state.account and state.accounts ??
+          accountAddress = (await state.externals.loadKeys()).find(({ name }) => name === localKeyName).address
```

Comment:
```suggestion
          localAddress = (await state.externals.loadKeys()).find(({ name }) => name === localKeyName).address
```

Reply:
this is not the local address but the address used to sign in for. can also be a ledger address

Reply:
hmmm ok. but still seems like we're removing `account` as a concept.